### ISSUE TITLE
Improve accuracy

### ISSUE THREAD

**Issue by mhaeming** [Maxoz99/ANNs-TensorFlow#7]
Could we take any measures to bump up the accuracy above the required 35-40%? 

- We could train for more epochs
- Modify the network architecture
